cylc/flow/task_pool.py

```diff
@@ -1136,7 +1140,7 @@ def _reload_taskdefs(self) -> None:
                 else:
                     # Keep active orphaned task, but stop it from spawning.
                     itask.graph_children = {}
-                    LOG.warning(
```

Comment:
Agreed.

Reply:
> Do we actually need this message?

Need? Possibly - it can be useful to present evidence after the fact, probably. Perhaps it should go to debug?

Reply:
IMO we should keep the message. It shows that Cylc correctly identified and handled a live task who's task definition has been removed by the reload. That's a fairly unusual circumstance, with potentially unpleasant results if we get it wrong.

Reply:
I'm not so convinced of the importance of this, the the task will continue with its previous config. I think this is a common circumstance right (?), what, I can't think what unpleasant / unexpected results this could lead to?

But if you disagree (no need to state your argument, I don't want to haggle this), I'm happy to keep this on at the info level.